README.md

```diff
@@ -19,3 +19,11 @@ This project uses [Gradle](https://gradle.org/). It requires that Java is alread
 2. On the command line, run `./gradlew run` (or, on Windows, `gradlew run`).
 
 You should see the message `"Feature flag '<flag key>' is <true/false> for this context"`.
+
+Optional: If you wish to see analytic events generated by this sample application, change the SDK configuration to enable events.
+
+```java
+    LDConfig config = new LDConfig.Builder()
+      .events(Components.sendEvents())
+      .build();
```

Comment:
In your testing, do you need to wait before quitting for the events to show up, or does this just work?

Reply:
Just tested this and the event shows without further need for flush or delayed quitting